src/Symfony/Component/Process/Process.php

```diff
@@ -1621,4 +1621,9 @@ private function getDefaultEnv(): array
 
         return $env;
     }
+
+    public function getStartTime(): ?float
```

Comment:
i tend to prefer a non-nullable API, thus a throw if `!isStarted()`.

Reply:
yet it makes sense to return null when the process didn't start

Reply:
(but I'm fine with throwing a LogicException as some other methods do so)
Please group the method with other public ones, before private and protected.

Reply:
> i tend to prefer a non-nullable API, thus a throw if `!isStarted()`.
I tend to prefer nullable API - it allows greater range of use and is more comfortable IMHO.

Reply:
the issue is its still a nullable , even if the process is started. Thats the common case IMHO ... in this case you dont want to account for null all the time.